elixir/apps/api/test/api/client/channel_test.exs

```diff
@@ -279,6 +297,18 @@ defmodule API.Client.ChannelTest do
                ]
              } in resources
 
+      assert %{
+               id: internet_resource.id,
+               type: :internet,
+               gateway_groups: [
+                 %{
+                   id: gateway_group.id,
+                   name: gateway_group.name
+                 }
+               ],
+               can_be_disabled: false
```

Comment:
right now the name is `can_toggle` but I can change it

Reply:
I think `can_toggle` would a bit ambiguous for the API, please change it if it's not a ton of work